### ISSUE TITLE
Cloud Logs & Cloud Logs Routing Services missing in VPE-enabled services 

### ISSUE THREAD

**Issue by petrandreev** [ibm-cloud-docs/vpc#220]
### Which topic are you reporting about?

https://cloud.ibm.com/docs/vpc?topic=vpc-vpe-supported-services#vpe-enabled-supported-services

### What do you think needs to be updated?

Cloud Logs & Cloud Logs Routing Services missing in VPE-enabled services.

### Anything else?

Furthermore: the documentation how to connect to [Cloud Logs Routing](https://cloud.ibm.com/docs/logs-router?topic=logs-router-vpe-connection&interface=api#vpe-connection-setup) isn't correct (region: Frankfurt)-
- in the Cloud Console there is no such _service offering_ Cloud Logs Routing
- the [CRN of the Ingestion endpoint](https://cloud.ibm.com/docs/logs-router?topic=logs-router-vpe-connection-crn#vpe-connection-crn-ingestion) is not accepted by Terraform _apply_, error: 
```
{
  "Result": {
    "errors": [
      {
        "code": "bad_field",
        "message": "Service not found in global catalog",
        "target": {
          "name": "Crn",
          "type": "field",
          "value": "crn:v1:bluemix:public:logs-router:eu-de:::endpoint:ingester.private.eu-de.logs-router.cloud.ibm.com"
        }
      }
    ],
    "trace": "08040a81-35d6-4f8a-9546-c489e0b8f056"
  }
}
```

**Comment by austinmama** [comment]
Thank you for your feedback!

Created new issue for the topic in the logs-router repo: https://github.ibm.com/cloud-docs/logs-router/issues/173

**Comment by austinmama** [comment]
Waiting on confirmation about whether Cloud Logs Routing Services needs to be listed in addition to the existing **IBM Cloud Logs** entry.

**Comment by austinmama** [comment]
Confirmed that this issue was fixed in the logs-router repository by Vanadis Crawford.  Closing issue.
